apps/domain/test/e2e/openrouter-streaming.e2e.spec.ts

```
@@ -0,0 +1,511 @@
+      }),
+    );
+
+    // Add the global exception filter
+    const { AllExceptionsFilter } = await import(
+      "../../src/filters/http-exception.filter"
+    );
+    app.useGlobalFilters(new AllExceptionsFilter());
+
+    await app.init();
+    await app.getHttpAdapter().getInstance().ready();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)